spec/features/admin_can_create_an_item_spec.rb

```diff
@@ -11,7 +11,7 @@
                   password_confirmation: "password",
                   role: 1)
     }
-  it 'can create a new item' do
+    it 'can create a new item' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.